satpy/tests/reader_tests/test_eum_l2_bufr.py

```diff
@@ -0,0 +1,351 @@
+     'h': 35786400., 'proj': 'geos', 'units': 'm'},
+    348,
+    348,
+    (-5567999.998550739, -5567999.998550739, 5567999.994203017,  5567999.994203017)
+)
+
+
+AREA_DEF_MSG_FES_3km_ext = geometry.AreaDefinition(
+    'msg_seviri_fes_9km_ext',
+    'MSG SEVIRI Full Earth Scanning service area definition with 9 km resolution',
+    "",
+    {'a': 6378169., 'b': 6356583.8, 'lon_0': 0.0,
+     'h': 35785831., 'proj': 'geos', 'units': 'm'},
+    1238,
+    1238,
+    (-5571748.888268564, -5571748.888155806, 5571748.888155806, 5571748.888268564)
+)
+
+# Test data for mock file
+DATA = np.random.uniform(low=250, high=350, size=(128,))
+LAT = np.random.uniform(low=-80, high=80, size=(128,))
+LON = np.random.uniform(low=-38.5, high=121.5, size=(128,))
+
+os.environ['ECCODES_BUFR_MULTI_ELEMENT_CONSTANT_ARRAYS'] = "1"
+
+# Test cases dictionaries
+TEST_DATA = {'GIIBUFRProduct_20231027140000Z_00_OMPEFS03_MET10_FES_E0000': {
+                'platform_name': 'MSG3',
+                'spacecraft_number': '10',
+                'RectificationLongitude': 'E0000',
+                'ssp_lon': 0.0,
+                'area': AREA_DEF_MSG_FES_3km_ext,
+                'seg_size': 3,
+                'file_type': 'seviri_l2_bufr_gii',
+                'key': '#1#brightnessTemperature',
+                'resolution': 9000},
+             'ASRBUFRProd_20231022224500Z_00_OMPEFS03_MET10_FES_E0000': {
+                'platform_name': 'MSG3',
+                'spacecraft_number': '10',
+                'RectificationLongitude': 'E0000',
+                'ssp_lon': 0.0,
+                'area': AREA_DEF_MSG_FES,
+                'seg_size': 16,
+                'file_type': 'seviri_l2_bufr_asr',
+                'key': '#1#brightnessTemperature',
+                'resolution': 48000},
+             'ASRBUFRProd_20231023044500Z_00_OMPEFS02_MET09_FES_E0455': {
+                'platform_name': 'MSG2',
+                'spacecraft_number': '9',
+                'RectificationLongitude': 'E0455',
+                'area': AREA_DEF_MSG_IODC,
+                'ssp_lon': 45.5,
+                'seg_size': 16,
+                'file_type': 'seviri_l2_bufr_asr',
+                'key': '#1#brightnessTemperature',
+                'resolution': 48000},
+             'MSG2-SEVI-MSGASRE-0101-0101-20191106130000.000000000Z-20191106131702-1362128.bfr': {
+                'platform_name': 'MSG2',
+                'spacecraft_number': '9',
+                'RectificationLongitude': 'E0455',
+                'area': AREA_DEF_MSG_IODC,
+                'ssp_lon': 45.5,
+                'seg_size': 16,
+                'file_type': 'seviri_l2_bufr_asr',
+                'key': '#1#brightnessTemperature',
+                'resolution': 48000},
+             'W_XX-EUMETSAT-Darmstadt,IMG+SAT,MTI1+FCI-2-ASR--FD------BUFR_C_EUMT_20230623092246_L2PF_IV_20170410170000_20170410171000_V__C_0103_0000.bin': {
+                'platform_name': 'MTGi1',
+                'spacecraft_number': '24',
```

Comment:
what is this number?

Reply:
I need to set it to this value to get the correct satelliteIdentifier = 47 + spacecraft_number for FCI
There is a big jump in numbers after MSG

Reply:
right, had forgotten about that. Thanks!